Scenarios/CGCT/CGCT/DemoScenarioText.m

```diff
@@ -0,0 +1,204 @@
+//******************************************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#import "DemoScenarioText.h"
+#import <CoreGraphics/CoreGraphics.h>
+#import <CoreText/CoreText.h>
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+
+@implementation DemoScenarioText
+
+static NSString *patternName;
+
+- (NSString *)name {
+  return @"Text";
+}
+
+- (UIColor *)backgroundColor {
+  return [UIColor colorWithRed:.6 green:.85 blue:1 alpha:1];
+}
+
+static void drawPattern(void *info, CGContextRef context) {
+  UIImage *patternImage = [UIImage
```

Comment:
did you have to do this manually instead of using UIImage and `UIColor colorWithPatternImage:`?